src/App.tsx

```diff
@@ -86,8 +81,6 @@ const Content = () => {
 
   const { dialogAreaRef } = useDialogArea() ?? {};
 
-  const showChainTokenPage = complianceState === ComplianceStates.FULL_ACCESS || isStakingEnabled;
```

Comment:
just double checking if we're good with showing this page with all different compliance states!